<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="8" total="9" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-04-21T23:02:55 IST" name="Default suite" finished-at="2022-04-21T23:03:17 IST" duration-ms="21957">
    <groups>
    </groups>
    <test started-at="2022-04-21T23:02:55 IST" name="Default test" finished-at="2022-04-21T23:03:17 IST" duration-ms="21957">
      <class name="com.Testcases.TestScenario_001">
        <test-method is-config="true" signature="BeforeTestcaseRun()[pri:0, instance:com.Testcases.TestScenario_001@4a883b15]" started-at="2022-04-21T23:02:55 IST" name="BeforeTestcaseRun" finished-at="2022-04-21T23:02:59 IST" duration-ms="4600" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- BeforeTestcaseRun -->
        <test-method signature="Tc_010()[pri:0, instance:com.Testcases.TestScenario_001@4a883b15]" started-at="2022-04-21T23:02:59 IST" name="Tc_010" finished-at="2022-04-21T23:03:13 IST" duration-ms="14100" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [https://www.justrechargeit.com/SignUp.aspx] but found [https://www.justrechargeit.com/myaccount_directory.aspx]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [https://www.justrechargeit.com/SignUp.aspx] but found [https://www.justrechargeit.com/myaccount_directory.aspx]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at com.Testcases.TestScenario_001.Tc_010(TestScenario_001.java:157)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:673)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:220)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:945)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:193)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:808)
at org.testng.TestRunner.run(TestRunner.java:603)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:429)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:423)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:383)
at org.testng.SuiteRunner.run(SuiteRunner.java:326)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1249)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
at org.testng.TestNG.runSuites(TestNG.java:1092)
at org.testng.TestNG.run(TestNG.java:1060)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Tc_010 -->
        <test-method is-config="true" signature="AfterTestcaseRun()[pri:0, instance:com.Testcases.TestScenario_001@4a883b15]" started-at="2022-04-21T23:03:13 IST" name="AfterTestcaseRun" finished-at="2022-04-21T23:03:17 IST" duration-ms="3174" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- AfterTestcaseRun -->
      </class> <!-- com.Testcases.TestScenario_001 -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
